extensions/amp-video-docking/0.1/amp-video-docking.js

```diff
@@ -54,52 +68,80 @@ import {
 
 const TAG = 'amp-video-docking';
 
-/** @visibleForTesting @const {number} */
-export const MARGIN_MAX = 30;
+/** Ratio to viewport width to use when docked to corner. */
+export const CORNER_WIDTH_RATIO = 0.3;
```

Comment:
Where is this coming from?

Reply:
It's static and not configurable, it was defined already under a different name.